itests/deals_retry_deal_no_funds_test.go

```diff
@@ -178,69 +186,3 @@ func TestDealsRetryLackOfFunds_blockInPublishDeal(t *testing.T) {
 	case <-time.After(time.Second * 15):
 	}
 }
-
-func TestDealsRetryLackOfFunds_belowLimit(t *testing.T) {
```

Comment:
As far as I can tell, this was testing a bug? We should always allow retries, no matter how low our account balance is.

Reply:
I don't think that's true -- I believe we explicitly _don't_ auto-retry (can be manually triggered, or happens on restart, per discussion [here](https://github.com/filecoin-project/lotus/pull/7454#issue-1018338971)).

Is this test failing as a result of your change? It may just need to have its funds lowered to, like, 1 attoFIL.

Reply:
But yes, we never auto-retry. But that wasn't what this was testing. This was testing whether or not the deal _failed_ if we were below the limit.

Reply:
Yeah, you seem to be right...